pkg/controller/config/config_controller.go

```diff
@@ -280,6 +295,16 @@ func (r *ReconcileConfig) replayData(ctx context.Context, w *watch.Set) error {
 		for i := range u.Items {
 			syncKey := r.syncMetricsCache.GetSyncKey(u.Items[i].GetNamespace(), u.Items[i].GetName())
 
+			foundExcludedNamespace := false
+			for _, ns := range r.configMatchSet.ExcludedNamespaces[match.Sync] {
```

Comment:
Same comment here about preparing a map before the for loop to cut down on execution time.

Reply:
updated to map[Operation]map[string]bool